app/assets/javascripts/host_edit.js

```diff
@@ -6,6 +6,12 @@ $(document).on('ContentLoad', function() {
 $(document).on('AddedClass', function(event, link) {
   load_puppet_class_parameters(link);
 });
+$(document).on('change', '#host_hostgroup_id', function(evt) {
+  hostgroup_changed(evt.target);
+});
+$(document).on('change', '#host_compute_resource_id, #compute_profile', function(evt) {
```

Comment:
are we certain there are no other matching elements anywhere in foreman or any of the plugins?

Reply:
should be as it is purpose of ID, but fair point... I can find some unique parent.